caluma/caluma_data_source/tests/data_sources.py

```diff
@@ -91,3 +93,30 @@ def get_data(self, user, question, context):
             label += f" foo: {context['foo']}"
 
         return [[slug, label]]
+
+
+class MyDataSourceWithOnCopy(BaseDataSource):
+    info = "Data source using on copy for testing"
+
+    def get_data(self):
+        return [
+            ["169cac4c-ab46-4521-bdba-4385f26ffb3", "label1"],
+            ["34948dc6-4b16-4a72-890b-cdbdc7da9f2", "label2"],
+            ["0935c1bc-5ff3-44a4-9089-03eac8872b0", "label3"],
+        ]
+
+    def get_change_value(self):
+        return ["changedv-4b16-4a72-890b-cdbdc7da9f2", "changed label2"]
+
+    def on_copy(
+        self, old_answer: Answer, new_answer: Answer, old_value: Tuple[str, str]
+    ) -> Tuple[Optional[str], Optional[str]]:
+        old_slug, _ = old_value
```

Comment:
I know it's "just" a test, but as those often are used as examples, I'd like to see a docstring here as well